docs/pages/2020_Intro_Week/sections/workflows_basic.rst

```diff
@@ -4,6 +4,30 @@
 Workflows: Basics
 *****************
 
+.. warning::
+
+    In order to launch the workflows of this section, we will be using the computers and codes set up in the first two hands-on sessions.
+    So, you will have to set the profile that you set up at the start of the tutorial as the default.
+    You can do this using:
+
+    .. code-block:: console
+
+        $ verdi profile setdefault <PROFILE_NAME>
+        Success: <PROFILE_NAME> set as default profile
+
+    Where <PROFILE_NAME> is the name of the profile you have set up (e.g. ``quicksetup`` is the default).
+    In order to check you have the required codes, you can use:
```

Comment:
```suggestion
        $ verdi profile setdefault quicksetup
        Success: quicksetup set as default profile

    You should now have the following codes available:
```

Reply:
I think many participants will not have used the name `quicksetup` for the default profile.

Reply:
At least, of those I helped in breakout rooms, many had a different profile name. Why not leave the <PROFILE_NAME>? Do you think it is confusing?